kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -316,6 +316,15 @@ def run(self, labels: list, analysis_id=None):
                     runtime_environments = [
                         e["name"] for e in thoth_config["runtime_environments"]
                     ]
+                    managers = thoth_config.get("managers")
+                    disallow_run_times = managers[0].get("disallow", None)
+                    allowed_run_times = managers[0].get("allow", None)
+                    if disallow_run_times:
+                        runtime_environments = list(
+                            set(runtime_environments) ^ set(disallow_run_times)
+                        )
+                    if allowed_run_times:
+                        runtime_environments = allowed_run_times
```

Comment:
I think this would be better as set intersection `&`

Reply:
ack, done